io.github.spacingbat3.webcord.yml

```
@@ -0,0 +1,50 @@
+app-id: io.github.spacingbat3.webcord
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
```

```suggestion
  - --device=all
  - --talk-name=org.freedesktop.Notifications
```
Allow webcord to send notifications

---

I would rather not allow notifications since as of now clicking on the notification crashes the app. [it's an upstream issue](https://github.com/SpacingBat3/WebCord/issues/134)

---

Oh yeah then it should be kept disabled